### ISSUE TITLE
Rewrite ACK to accept 200 or 404 delivery

### ISSUE THREAD

**Issue by willyaranda** [telefonicaid/notification_server#128]
For example, if a notification is sent to a device and the device couldn't deliver to the application (because it had been uninstalled), the UA shall inform the NS with a 404 to remove it from the DB and avoid future notifications to it.

**Comment by frsela** [comment]
You mean the registerUA ACK?
This is the one to known about if the UA was registered before the command?

**Comment by willyaranda** [comment]
No, on message ACK, gecko should say if the delivery is correct or not. 200 if OK (implicit) and 404 if the delivery failed, i.e. the app has been uninstalled but not unregistered.

**Comment by frsela** [comment]
Oh! that's true ... sorry :p
